.vscode/launch.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Before commit I checked some MS repos for that it seems they commit launch.json and tasks.json. I think we can continue to ignore .vscode and specifically settings.json there. Thus I explicitly commited these files only.